[TTS] add CI test for RADTTS training recipe.

XuesongYang opened the pull request:
Signed-off-by: Xuesong Yang <1646669+XuesongYang@users.noreply.github.com>

# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.


## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

borisfom reviewed (changes requested):
I have updated my branch ... a lot, please pick up - or should we merge mine first ?

XuesongYang commented:
> I have updated my branch ... a lot, please pick up - or should we merge mine first ?

This is a draft PR in order to test radtts. Since it is contingent on your fix, do you mind if cherry-pick this one to your PR? I will let you know once I cleaned up this PR. Thanks.

XuesongYang commented:
close this PR since https://github.com/NVIDIA/NeMo/pull/5358 has cherry-picked these changes.
